images/runtime/python/generateDockerfiles.sh

```diff
@@ -8,24 +8,28 @@ set -e
 
 declare -r DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 declare -r PYTHON_VERSIONS_PATH=$DIR/../../../build/__python-versions.sh
-declare -r VERSIONS_FILE="$DIR/pythonVersions.txt"
 declare -r DOCKERFILE_TEMPLATE="$DIR/Dockerfile.template"
-declare -r IMAGE_NAME_PLACEHOLDER="%PYTHON_BASE_IMAGE%"
-declare -r IMAGE_SUFFIX="-slim-stretch"
+declare -r PYTHON_MAJOR_VERSION_PLACEHOLDER="%PYTHON_MAJOR_VERSION%"
+declare -r PYTHON_VERSION_PLACEHOLDER="%PYTHON_VERSION%"
+declare -r PYTHON_FULL_VERSION_PLACEHOLDER="%PYTHON_FULL_VERSION%"
```

Comment:
I am a bit confused with this line and the above...could you add a comment as to how they would like?